internal/webapi/server.go

```diff
@@ -62,13 +57,17 @@ func NewServer(db database.Database, auth authentication.Authenticator[APIAuthCr
 	femto.OnPost(mux, "/api/admin/add_workstation", api.addMachine)
 	femto.OnPost(mux, "/api/admin/stats/modify", api.modifyMachineInfo)
 	femto.OnPost(mux, "/api/admin/rm_workstation", api.removeMachine)
-	femto.OnGet(mux, "/api/admin/confirm", api.confirmAdmin)
+	femto.OnGet(mux, "/api/admin/confirm", func(r *http.Request, l *slog.Logger) (*femto.Response[UsernameReminder], error) {
+		return api.confirmAdmin(auth, r, l)
```

Comment:
NIT: You should probably make `api.confirmAdmin` just follow the expected signature, so we don't need this ugly lambda.

(This probably meeds adding `auth` to `api`)

Reply:
I just followed the pattern from `"/api/admin/auth"`. I think changing this should be a follow-up PR